src/PlugWallet/index.ts

```diff
@@ -50,6 +51,10 @@ class PlugWallet {
     this.name = val;
   }
 
+  public async sign(payload: BinaryBlob): Promise<BinaryBlob> {
+    return this.identity.sign(payload);
```

Comment:
@qti3e changed what you had done here. Janison told me that it was a performance thing. Can you let me know if that persists here? Instead of receiving an `ArrayBuffer` I'm now directly receiving a Blob